README.md

```diff
@@ -137,7 +137,152 @@ This now has created the following structure, either on disk or as an in memory
 
 ## API
 
-See https://ipfs.github.io/js-ipfs-repo
+### Setup
+
+#### new Repo(path[, options])
+
+Creates an IPFS Repo.
+
+Arguments:
+
+* `path` (string, mandatory): the path for this repo
+* `options` (object, optional): may contain the following values
+  * `lock` (string, defaults to `"fs"` in Node.js, `"memory"` in the browser): what type of lock to use. Lock has to be acquired when opening.
+  * `storageBackends` (object, optional): may contain the following values, which should each be a class implementing the [datastore interface](https://github.com/ipfs/interface-datastore#readme):
+    * `root` (defaults to [`datastore-fs`](https://github.com/ipfs/js-datastore-fs#readme) in Node.js and [`datastore-level`](https://github.com/ipfs/js-datastore-level#readme) in the browser)
+    * `blocks` (defaults to [`datastore-fs`](https://github.com/ipfs/js-datastore-fs#readme) in Node.js and [`datastore-level`](https://github.com/ipfs/js-datastore-level#readme) in the browser)
+    * `datastore` (defaults to `datastore-level`)
+
+```js
+const repo = new Repo('path/to/repo')
+```
+
+#### repo.init (callback)
+
+Creates the necessary folder structure inside the repo.
+
+#### repo.open (callback)
+
+Locks the repo.
+
+#### repo.close (callback)
+
+Unlocks the repo.
+
+#### repo.exists (callback)
+
+Tells whether this repo exists or not. Callsback with `(err, bool)`.
+
+### Repos
+
+Root repo:
+
+#### repo.put (key, value:Buffer, callback)
```

Comment:
Unnecessary spaces.

Also, can we put these between ``, helps readability on the readme.